corruption_eda_w203.Rmd

```diff
@@ -39,8 +39,10 @@ First, we load some of the packages we will need for the analysis and the data i
 
 Load the data:
 ```{r, echo=TRUE}
+install.packages('pacman')
 source("utils/functions.R")
-df <- load_rda('data/Corrupt.Rdata')
```

Comment:
We added a function that does this, I think we should either print the functions separately or move them back here for the submission

Reply:
yep, totally fine with going back to how you and Erik did it. I did this separately because I couldn't get it to work implicitly in my environment